components/roadmap-grid/header.tsx

```diff
@@ -19,11 +19,11 @@ export default function Header({
   }
 
   return (
-    <>
+    <Skeleton isLoaded={!globalLoadingState.get()} >
       <Flex direction={'row'} lineHeight={1} >
```

Comment:
since we're using globalLoadingState for all the skeleton loading, it's kind of a prettier spinner. Can we target the specific data necessary to render each component instead of the globalLoadingState?

```suggestion
  const isReadyForDisplay = issueDataState.ornull?.title.value && issueDataState.ornull?.html_url.value
  return (
    <Skeleton isLoaded={!isReadyForDisplay} >
      <Flex direction={'row'} lineHeight={1} >
```

Reply:
we can tackle this in a later PR; can you create an issue to track that? in my mind, the skeleton loading was intended to also encapsulate the partial rendering of content, but this PR makes sense as an upgrade to the existing spinner UI